tests/test_core.py

```diff
@@ -627,9 +692,10 @@ class MyClass(NSObject):  # noqa: F811
     def test_interface(self):
         "An ObjC protocol implementation can be defined in Python."
 
+        Callback = ObjCProtocol('Callback')
         results = {}
 
-        class Handler(NSObject):
+        class Handler(NSObject, protocols=[Callback]):
```

Comment:
Yep, they are really useful for this sort of thing. I didn't want to do it this way originally - first I tried to put the protocols directly in the bases list (after the base class), but Python didn't like that, because the superclass has a different metaclass (`ObjCClass`) than the protocols (`ObjCProtocol`). In the end this way is probably best though, it's easier to read and cleaner to implement.